.github/workflows/build_and_test.yml

```diff
@@ -49,17 +51,32 @@ jobs:
           - image: alpine315
             python: python3
             config-zstd: OFF
+          - image: alpine317
+            python: python3
+            config-zstd: ON
+            clang-tidy: ON
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Configure PcapPlusPlus
+      - if: ${{ matrix.clang-tidy }}
+        name: Configure PcapPlusPlus w/ Clang-Tidy
+        run: CXX=clang++ CC=clang cmake -DLIGHT_PCAPNG_ZSTD=${{ matrix.config-zstd }} -DPCAPPP_ENABLE_CLANG_TIDY=ON -S . -B $BUILD_DIR
```

Comment:
All we want is to build PcapPlusPlus with `clang-tidy`, no?
In that case, maybe it'd be nicer to add another `job` that just builds PcapPlusPlus without running the tests etc. I think it's a simpler solution than having this `if`... what do you think?